packages/react-hig/src/elements/components/GlobalNav/StatefulTabs.js

```diff
@@ -0,0 +1,94 @@
+/**
+Copyright 2016 Autodesk,Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+import React from 'react';
+
+import Tabs2 from './Tabs2';
+import Tab2 from './Tab2';
+
+class StatefulTab extends React.Component {
+  // This is never actually rendered
+  render() {
+    return <div />;
+  }
+}
```

Comment:
Because the end user doesn't actually interact with Tab2 at all in the child nodes we just need a proxy which represents those nodes. 

We could export Tab2 directly but then we'd need to use React.cloneElement to add the onClick and active properties which is pretty janky. 

This is nice and clean and allows us to expose only the APIs that we want.